Add documentation for requesting a new group on the cluster (#1208)
rug-cit-hpc/league-of-robots: roles/online_docs/templates/mkdocs/docs/new_group.md

#jinja2: trim_blocks:True
# How to get a new group on the cluster

> A department head or principal investigator can request a new group on the cluster(s).

Reviewer: ```suggestion
(if employed @UMCG) A department head or principal investigator can request a new group on the cluster(s).
```

Author: We also have requests from outside UMCG, so not only if you're employed @UMCG

The information needed to request a new group, includes:

 * Name of the group: umcg-[groupname] -  **NAME IS SET IN STONE!**
 * The names and mailing addresses of the:
    * Group owner(s)
    * Datamanager(s)
    * Group member(s)

> For each of the persons mentioned above, we need a confirmation from the secretary on the enddate of their contract with the UMCG.

Reviewer: ```suggestion
For each of the persons mentioned above, we need a confirmation either on the enddate of their contract.
(if employed @UMCG, department's secretary should send en end date **directly** to the helpdesk)
```

## Minimal requirements

The minimal requirements for a main group are as follows:

 * Group leaders / PIs can request new main groups. When the main group is created they will be registered as the 
group owners.
 * Group owners are responsible for
    * Processing (accepting or rejecting) requests for group membership.
    * Securing funding and paying the bills.
    * Appointing data managers for their group.
    * Approving user membership.
 * Data managers are responsible for the group's data on ```prm```, ```rsc``` (if available) and ```arc``` (if 
available) storage systems and
    * Ensure the group makes arrangements what to store how and where. E.g file naming conventions, file formats 
to use, etc.
    * Enforce the group's policy on what to store how and where by reviewing data sets produced by other group 
members on ```tmp``` file systems before migrating/copying them to ```prm``` or ```arc``` (if available).

>     * Can put released versions of data sets on ```rsc``` storage, so it can be used as reference data by alle 
> members of the group.

Reviewer: ```suggestion
    * Storing released versions of data sets on ```rsc``` storage, so it can be used as reference data by entire group.
```
